tests/test_config.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+
+from flask import current_app
+from flask_testing import TestCase
+
+from bucketlist import app
+
+
+class TestDevelopmentConfig(TestCase):
```

Comment:
I don't see the need for this whole file seeing that you have a testing config class in your config file.